controllers/component_build_controller.go

```diff
@@ -33,11 +33,12 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/metrics"
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
 
-	"github.com/go-logr/logr"
+	ctrllog "sigs.k8s.io/controller-runtime/pkg/log"
 
 	"github.com/prometheus/client_golang/prometheus"
 	appstudiov1alpha1 "github.com/redhat-appstudio/application-api/api/v1alpha1"
 	"github.com/redhat-appstudio/build-service/pkg/boerrors"
+	l "github.com/redhat-appstudio/build-service/pkg/logs"
```

Comment:
Give it a more meaningful name instead of `l`?

Reply:
I have that short name because it's used several times in a line and used in about 2/3 of logs statements. Having it long could make it sort of awkward for reading. Also, short one-two latter names are quite widespread in golang.

Reply:
But if you insist on renaming to, say, `logconsts`, please let me know.